### PR TITLE
[hevce] prevent HRD parameters submitting for CQP cases

### PR DESCRIPTION
**Pull request by vilichev**
HRD parameters cause switching to CBR in media driver for HEVCe FEI

### CODE REVIEW

**Review by lakulako — COMMENTED** [review]
AddVaMiscHRD already considers CQP case. Please check why it doesn't work. Anyway better to avoid doubled check.

**Comment by vilichev** [comment]
@lakulako AddVaMiscHRD just sets HRD parameters depending on CQP. But we need to avoid submitting of such buffer to driver for CQP case.

**Comment by lakulako** [comment]
@vilichev, may be it would be more correct not to switch to CBR if HRD buffer is attached. Anyway CBR is not the only mode that uses HRD.

File: _studio/mfx_lib/encode_hw/hevc/linux/base/hevcehw_base_va_packer_lin.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019-2020 Intel Corporation
+D/ Copyright (c) 2019-2020 Intel Corporation
```
Comment (lakulako) [inline _studio/mfx_lib/encode_hw/hevc/linux/base/hevcehw_base_va_packer_lin.cpp:1]: D/  - misprint?

**Comment by vilichev** [comment]
> @vilichev, may be it would be more correct not to switch to CBR if HRD buffer is attached. Anyway CBR is not the only mode that uses HRD.
@lakulako I think we should fix it in MSDK too, since HRD does not intend for CQP/ICQ cases.

File: _studio/mfx_lib/encode_hw/hevc/linux/base/hevcehw_base_va_packer_lin.cpp (hunk 1 above)
Comment (vilichev) [reply _studio/mfx_lib/encode_hw/hevc/linux/base/hevcehw_base_va_packer_lin.cpp]: Thanks! Done

**Review by saosipov — APPROVED** [review]

**Review by lakulako — APPROVED** [review]

**Review by dmitryermilov — APPROVED** [review]

**Pull request merged**